index.html

```diff
@@ -39,152 +47,158 @@ <h3 class="subject_text">I'm a Software Developer</h3>
-                industry's standard.</p>
+        <div class="feature_project">
+            <div class="feature-project_preview_image"></div>
+            <div class="feature_project_info">
+                <div class="feature-project_title">
+                    <h3>Architecture</h3>
+                </div>
+                <div class="feature-project_description_container">
+                    <p class="feature-project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a standard dummy text.</p>
+                </div>
+                <ul class="feature-project_skills">
+                    <li class="feature-skill_container"><span class="feature-skill">CSS</span></li>
+                    <li class="feature-skill_container"><span class="feature-skill">HTML</span></li>
+                    <li class="feature-skill_container"><span class="feature-skill">Bootstrap</span></li>
+                    <li class="feature-skill_container"><span class="feature-skill">Ruby</span></li>
+                </ul>
+                <button type="button" class="feature-project_link">
+                    <span class="feature-link_text">See project</span>
+                </button>
             </div>
-            <div class="project_image"></div>
-            <ul class="project_skills">
-                <li class="skill_container"><span class="skill">HTML</span></li>
-                <li class="skill_container"><span class="skill">Bootstrap</h4></li>
-                <li class="skill_container"><span class="skill">Ruby</span></li>
-            </ul>
-            <button type="button" class="project_link"><span class="link_text">See project</span></button>
         </div>
-        <div class="project_container_4">
-            <div class="project_title">
-                <h3>Business Website</h3>
+        <div class="portfolio_thumbnails">
+            <div class="project_container_1">
+                <div class="project_title">
+                    <h3>E-commerce Website</h3>
+                </div>
+                <div class="project_description">
+                    <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. Has been the 
+                    industry's standard.</p>
+                </div>
+                <div class="project_image"></div>
+                <ul class="project_skills">
+                    <li class="skill_container"><span class="skill">HTML</span></li>
+                    <li class="skill_container"><span class="skill">Bootstrap</span></li>
+                    <li class="skill_container"><span class="skill">Ruby</span></li>
+                </ul>
+                <button type="button" class="project_link"><span class="link_text">See project</span></button>
```

Comment:
[Optional] A close look at the figma design for the cards shows two displays for the cards. The first card with the "See Project" button shows how each card should look when hovered while the remaining cards without the "See Project" button shows how the cards will look without a hover.

Project card when hovered:
![image](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/20554583/2b825e3f-3764-4885-9d7f-9d8587bc9b74)

Project card without hover:
![image](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/20554583/a2494b72-91b5-490d-bfbd-8b9d96f622d0)